### ISSUE TITLE
OSError: could not get source code

### ISSUE THREAD

**Issue by sobassy** [scrapy/scrapy#4906]
I tried obfuscate scrapy projects.

in scrapy==1.8.0, I could obfuscate this, and exited gracefully.
but in scrapy==2.2.0,  I could obfuscate this, and caused this Error: OSError: could not get source code
I use pyarmor for obfuscate.

I checked inspect.py, and debug;

in inspect.py line 786;
```
# raise OSError('could not get source code')
raise OSError('could not get source code', file)
```

output is 

```
OSError: [Errno could not get source code] <frozen spiders.test>
```

test is my spider's name.

so I thought When read obfuscated file recursively, cannot load frozen object.

If you have any advices, please give me any solutions. thank you

**Comment by wRAR** [comment]
If you think this is a problem with Scrapy you need to provide steps to reproduce the problem.

**Comment by sobassy** [comment]
Thank you for your reply.

This error is fixed by using scrapy==1.8.0.
and I think this error occurred in obfuscating process.

I need to study for pyarmor and obfuscate more.

Thank for your advice :)

**Comment by colab-site** [comment]
[PT-BR] Solução:
https://stackoverflow.com/questions/61478001/scrapy-pyinstaller-oserror-could-not-get-source-code-twisted-internet-defer
